meilisearch-http/src/routes/index.rs

```diff
@@ -390,19 +390,10 @@ pub async fn get_all_updates_status(ctx: Context<Data>) -> SResult<Response> {
 
 pub async fn delete_index(ctx: Context<Data>) -> SResult<StatusCode> {
     ctx.is_allowed(IndexesWrite)?;
+    let _ = ctx.index()?;
```

Comment:
Ok, so now the 404 is raised by [the index method](https://github.com/meilisearch/MeiliSearch/blob/bd51bec397b2e9ecc7377e2fd42a53a4d0753af4/meilisearch-http/src/helpers/tide.rs#L93-L101).